README.md

```diff
@@ -1,48 +1,188 @@
+      </li>
+      </ul>
+    </td>
+  </tr>
+
+  <tr>
+    <td><a href="https://docs.wavefront.com/wavefront_sdks.html#sdks-that-instrument-frameworks">Framework SDK</a></td>
+    <td align="justify">Reports predefined traces, metrics, and histograms from the APIs of a supported app framework. Lets you get started quickly with minimal code changes.</td>
+    <td>
+      <ul>
+      <li><b>Java</b>:
+      <a href="https://github.com/wavefrontHQ/wavefront-dropwizard-sdk-java">Dropwizard</a> <b>|</b> <a href="https://github.com/wavefrontHQ/wavefront-gRPC-sdk-java">gRPC</a> <b>|</b> <a href="https://github.com/wavefrontHQ/wavefront-jaxrs-sdk-java">JAX-RS</a> <b>|</b> <a href="https://github.com/wavefrontHQ/wavefront-jersey-sdk-java">Jersey</a></li>
+      <li><b>.Net/C#</b>: 
+      <a href="https://github.com/wavefrontHQ/wavefront-aspnetcore-sdk-csharp">ASP.Net core</a> </li>
+      <!--- [Python](wavefront_sdks_python.html#python-sdks-that-instrument-frameworks) --->
+      </ul>
+    </td>
+  </tr>
+
+  <tr>
+    <td><a href="https://docs.wavefront.com/wavefront_sdks.html#sdks-for-sending-raw-data-to-wavefront">Sender SDK</a></td>
+    <td align="justify">Lets you send raw values to Wavefront for storage as metrics, histograms, or traces, e.g., to import CSV data into Wavefront. 
+    </td>
+    <td>
+      <ul>
+      <li>
+      <b>Java</b>: <a href ="https://github.com/wavefrontHQ/wavefront-sdk-java">Sender SDK</a>
+      </li>
+      <li>
+      <b>Python</b>: <a href ="https://github.com/wavefrontHQ/wavefront-sdk-python">Sender SDK</a>
+      </li>
+      <li>
+      <b>Go</b>: <a href ="https://github.com/wavefrontHQ/wavefront-sdk-go">Sender SDK</a>
+      </li>
+      <li>
+      <b>.Net/C#</b>: <a href ="https://github.com/wavefrontHQ/wavefront-sdk-csharp">Sender SDK</a>
+      </li>
+      <li>
+      <b>C++</b>: <a href ="https://github.com/wavefrontHQ/wavefront-sdk-cpp">Sender SDK</a>
+      </li>
+      </ul>
+    </td>
+  </tr>
+</table> 
+
+## Prerequisites
+
+* Go 1.9 or higher
+* Import Wavefront packages.
+    ```go
+    import (
+    	"github.com/wavefronthq/wavefront-opentracing-sdk-go/reporter"
+    	"github.com/wavefronthq/wavefront-opentracing-sdk-go/tracer"
+    	"github.com/wavefronthq/wavefront-sdk-go/application"
+    	"github.com/wavefronthq/wavefront-sdk-go/senders"
+    )
+    ```
+
+## Usage
+
+[Tracer](https://github.com/opentracing/specification/blob/master/specification.md#tracer) is an OpenTracing [interface](https://github.com/opentracing/opentracing-go#api-overview-for-those-adding-instrumentation) for creating spans and propagating them across arbitrary transports.
 
-## Requirements and Installation
+This SDK provides a `WavefrontTracer` that implements the `Tracer` interface. A `WaverfrontTracer`:
+* Creates spans and sends them to Wavefront.
+* Automatically generates and reports [RED metrics](https://github.com/wavefrontHQ/wavefront-opentracing-sdk-java/blob/master/docs/metrics.md) from your spans.
```

Comment:
We need a similar page for the Go SDK too, right: https://github.com/wavefrontHQ/wavefront-opentracing-sdk-java/blob/master/docs/metrics.md?

Will the same content work?